depends/patches/qt/force_bootstrap.patch

```diff
@@ -0,0 +1,20 @@
+Qt lrelease tool depends on XML features.
+No need to build the libQt5Xml module if tools are
+bootstrapped, even if not cross-compiling.
+
+We should be able to drop this once we are using Qt 5.15:
+ - QTBUG-76177
+ - upstream commit: 9e6943370585bdc791caae6c83555b3aa5efe063
```

Comment:
Where does this commit come from?

```bash
# git remote -v
origin	https://code.qt.io/cgit/qt/qtbase.git (fetch)
origin	https://code.qt.io/cgit/qt/qtbase.git (push)

# git branch --contains 9e6943370585bdc791caae6c83555b3aa5efe063
error: no such commit 9e6943370585bdc791caae6c83555b3aa5efe063
```

If I look for one of the actual deprecation commits i.e [82d02b7b95908dec16e41c5af1c63579729c589b](https://codereview.qt-project.org/gitweb?p=qt%2Fqtbase.git;a=commit;h=82d02b7b95908dec16e41c5af1c63579729c589b):
```bash
# git branch --contains 82d02b7b95908dec16e41c5af1c63579729c589b
  5.15
  5.15.0
  5.15.1
  5.15.2
  6.0
  6.0.0
  6.1
  dev
```

Reply:
Thanks for checking!
Sorry, I don't know what was in my clipboard at that time :man_facepalming: 

Indeed, you are pointing at the right commit https://github.com/qt/qtbase/commit/82d02b7b95908dec16e41c5af1c63579729c589b

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/21589#issuecomment-839582392).